assets/js/payment-method-extensions/payment-methods/stripe/credit-card/use-payment-processing.js

```diff
@@ -131,14 +133,15 @@ export const usePaymentProcessing = (
 					);
 				}
 				setSourceId( response.source.id );
+				const savePaymentMethodKey = `wc-${ PAYMENT_METHOD_NAME }-new-payment-method`;
```

Comment:
👍 nice that this is all that is needed. However it does make me wonder since the savePaymentMethodKey can be dynamically arrived, we should probably explore shifting the saved payment method checkbox component _outside_ of payment methods all-together and have checkout handle it. That will remove one more thing from payment methods to have to worry about and gives us tighter control over the behaviour of this checkbox.